### PR TITLE
App Config Store Partial Load Fix

### PR DESCRIPTION
**Pull request by mrm9084**
## Description
Fixes a bug that allowed stores using both /application/ and /<app_name>/ to load when one of them fails, but then stopped refresh from triggering on that store.

### CODE REVIEW

File: spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AppConfigurationPropertySourceLocator.java [hunk 1]
```diff
@@ -137,28 +137,35 @@ private void addPropertySource(CompositePropertySource composite, ConfigStore st
...
                 } else {
-                    LOGGER.warn("Unable to load configuration from Azure AppConfiguration Service for " + sourceContext,
+                    LOGGER.warn(
+                            "Unable to load configuration from Azure AppConfiguration store " + store.getEndpoint()
+                                    + ". The configurations starting with " + sourceContext + "failed to load.",
                             e);
                     StateHolder.setLoadState(store.getEndpoint(), false);
                 }
+                // If anything breaks we skip out on loading the rest of the store.
+                return;
```
Comment (saragluna) [inline spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AppConfigurationPropertySourceLocator.java:165]: If we return here, we'll not be able to load the store.

Comment (mrm9084) [reply spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AppConfigurationPropertySourceLocator.java:165]: That is the idea. If a user uses both /application/ and /application_name/. One can load while the other does not. This causes confusion and error.

Comment (mrm9084) [reply spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AppConfigurationPropertySourceLocator.java:165]: That is the idea. If a user uses both /application/ and /application_name/. One can load while the other does not. This causes confusion and error.

**Comment by saragluna** [comment]
Please help resolve the conflicts.

**Pull request merged**